### PR TITLE
Added new documentation : [GCP GKE Tracing - Application and eBPF]

### PR DESCRIPTION
**Pull request by sudhanshu456**

### CODE REVIEW

File: data/docs/gcp-monitoring/gke-tracing.mdx [hunk 1]
```diff
@@ -0,0 +1,1190 @@
...
+
+4. Confirm that the necessary Kubernetes resources are created:
+
+    ```sh
+    kubectl get pods,services,configmaps -n platform
+    ```
+
+    Check if the required pods, services, and config maps are running and in a healthy state.
+
+5. Check if all the Pixie deployments are in the correct order:
```
Comment (Jeet-14) [inline data/docs/gcp-monitoring/gke-tracing.mdx:1143]: Do we need pixie for app tracing?
Maybe we can add 


5. Check logs of signoz:
Depends on how you have installed signoz, check logs of service/ docker container/ pods.

and let Pixie as point 6 if needed!

File: data/docs/gcp-monitoring/gke-tracing.mdx [hunk 2]
```diff
@@ -0,0 +1,1190 @@
...
+
+4. Verify network connectivity:
+
+    * Ensure that the GKE cluster has network access to the SigNoz ingestion endpoint (Self-Hosted SigNoz VM IP:4317).
+    * Check if there are any network security groups or firewalls blocking the required ports.
+
+5. Check the SigNoz Self-Host VM
+    * Confirm that all the SigNoz related services are running properly on the VM or not.
+
+6. Check if all the Pixie deployments are in the correct order:
```
Comment (Jeet-14) [inline data/docs/gcp-monitoring/gke-tracing.mdx]: AFAIK, we were not running pixie on VM, so it is not needed for app tracing. Can you confirm once with sumedh as well?

File: data/docs/gcp-monitoring/gke-tracing.mdx [hunk 3]
```diff
@@ -0,0 +1,1190 @@
...
+    ports:
+    - name: flask
+      containerPort: 5002
+    env:
+    - name: MONGO_HOST
+      value: localhost
+    - name: OTEL_RESOURCE_ATTRIBUTES
+      value: service.name=sample-app
+    - name: OTEL_EXPORTER_OTLP_ENDPOINT
+      value: http://<app-fqdn>:4317
```
Comment (Jeet-14) [inline data/docs/gcp-monitoring/gke-tracing.mdx]: [app-fqdn](http://kubelet-otel-k8s-infra-otel-agent.signoz.svc.cluster.local:4317/) it is hyperlink (in G-Doc)
But here it is not.

File: data/docs/gcp-monitoring/gke-tracing.mdx [hunk 4]
```diff
@@ -0,0 +1,1190 @@
...
+    ports:
+    - name: flask
+      containerPort: 5002
+    env:
+    - name: MONGO_HOST
+      value: localhost
+    - name: OTEL_RESOURCE_ATTRIBUTES
+      value: service.name=sample-app
+    - name: OTEL_EXPORTER_OTLP_ENDPOINT
+      value: http://kubelet-otel-k8s-infra-otel-agent.signoz.svc.cluster.local:4317
```
Comment (Jeet-14) [inline data/docs/gcp-monitoring/gke-tracing.mdx]: Need to change this to  ```<IP Address of the VM>:4317```

File: data/docs/gcp-monitoring/gke-tracing.mdx [hunk 5]
```diff
@@ -0,0 +1,1190 @@
...
+opentelemetry-exporter-otlp==1.22.0 
+``` 
+
+Install those packages in dockerfile configuration.
+
+```Dockerfile 
+… 
+# install dependencies
+RUN pip install -r requirements.txt
+RUN opentelemetry-bootstrap --action=install…
```
Comment (Jeet-14) [inline data/docs/gcp-monitoring/gke-tracing.mdx]: '...' should be in new line not after install keyword, wanted to indicated that we've truncated  Dockerfile, it is not full file

File: data/docs/gcp-monitoring/gke-tracing.mdx [hunk 6]
```diff
@@ -0,0 +1,1190 @@
...
+```yaml
+apiVersion: v1
+kind: ConfigMap
+  name: kubelet-otel-k8s-infra-otel-agent
+data:
+  otel-agent-config.yaml: |2-
+
+    exporters:
+      otlp:
+        endpoint: "<App-endpoint>"
```
Comment (Jeet-14) [inline data/docs/gcp-monitoring/gke-tracing.mdx]: We;ll need same placeholder here and https://github.com/SigNoz/signoz-web/pull/599/files#diff-0d976b600f5acc59d180ed23cab10aabf1efc7fbe8e9294890c9782b2a8ce153R1082-R1083

File: data/docs/gcp-monitoring/gke-tracing.mdx [hunk 7]
```diff
@@ -0,0 +1,1190 @@
...
+**Step 1:** Add the SigNoz helm repo using this command
+
+```sh
+helm repo add SigNoz [https://charts.signoz.io](https://charts.signoz.io)
+```
+
+<Tabs>
+<TabItem value="self-hosted-on-gke-cluster" label="For Self-hosted SigNoz in GKE Cluster" default>
+
+**Step 2:** Install the OTel Collector agent in GKE where SigNoz is running in a Self-hosted VM.
```
Comment (Jeet-14) [inline data/docs/gcp-monitoring/gke-tracing.mdx]: Should be : Install the OTel Collector agent in GKE where SigNoz is running in a GKE cluster

File: data/docs/gcp-monitoring/gke-tracing.mdx (hunk 3 above)
Comment (sudhanshu456) [reply data/docs/gcp-monitoring/gke-tracing.mdx]: why it would be hyperlink I didn't get you?

Comment (sudhanshu456) [reply data/docs/gcp-monitoring/gke-tracing.mdx]: It is yaml right?

File: data/docs/gcp-monitoring/gke-tracing.mdx (hunk 4 above)
Comment (sudhanshu456) [reply data/docs/gcp-monitoring/gke-tracing.mdx]: I think we didn't mention it right on the doc; I converted from that, so it better correct it there as well.

File: data/docs/gcp-monitoring/gke-tracing.mdx (hunk 2 above)
Comment (sudhanshu456) [reply data/docs/gcp-monitoring/gke-tracing.mdx]: @sum3dh

File: data/docs/gcp-monitoring/gke-tracing.mdx (hunk 7 above)
Comment (sudhanshu456) [reply data/docs/gcp-monitoring/gke-tracing.mdx]: will fix it

File: data/docs/gcp-monitoring/gke-tracing.mdx (hunk 2 above)
Comment (Sum3dh) [reply data/docs/gcp-monitoring/gke-tracing.mdx]: Yes, we can skip this part for app tracing.

File: data/docs/gcp-monitoring/gke-tracing.mdx (hunk 5 above)
Comment (Jeet-14) [reply data/docs/gcp-monitoring/gke-tracing.mdx]: fixed

File: data/docs/gcp-monitoring/gke-tracing.mdx (hunk 4 above)
Comment (Jeet-14) [reply data/docs/gcp-monitoring/gke-tracing.mdx]: fixed

File: data/docs/gcp-monitoring/gke-tracing.mdx (hunk 6 above)
Comment (Jeet-14) [reply data/docs/gcp-monitoring/gke-tracing.mdx]: fixed

File: data/docs/gcp-monitoring/gke-tracing.mdx (hunk 3 above)
Comment (Jeet-14) [reply data/docs/gcp-monitoring/gke-tracing.mdx]: fixed

File: data/docs/gcp-monitoring/gke-tracing.mdx (hunk 2 above)
Comment (sudhanshu456) [reply data/docs/gcp-monitoring/gke-tracing.mdx]: added a note

**Review by Calm-Rock — APPROVED** [review]

**Review by yuvraajsj18 — APPROVED** [review]

**Pull request merged**